lochness/tree/__init__.py

```diff
@@ -23,6 +24,7 @@
     },
     'behav_qc': {
         'raw': Template('${base}/mri_behav/processed/behav_qc'),
+        'processed': Template('${base}/mri_behav/processed/behav_qc')
```

Comment:
Sorry about the silly question--the directories after `{base}` e.g. `mri_behave` will be created by `lochness`, right?

Reply:
Correct. (but `behav_qc` is not used for DPACC/ProNET/Prescient currently - may have to discuss about the structure when we come to use this data type.)